dotnet/simple-app/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿/*
+Copyright 2022 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Text;
+using ImmuDB;
+using ImmuDB.Exceptions;
+
+namespace simple_app;
+
+class Program
+{
+    public static async Task Main(string[] args)
+    {
+        await InitializeExample();
+        await AnotherInitializeExample();
+        await YetAnotherInitializeExample();
+        await MiscFunctionsUsageExample();
+        await ImmuClient.ReleaseSdkResources();
+    }
+
+    private static async Task InitializeExample()
+    {
+        var client = ImmuClient.NewBuilder().Build();
+        await client.Open("immudb", "immudb", "defaultdb");
+
+        string key = "hello";
+
+        try
+        {
+            // Setting (adding) a key-value.
+            await client.Set(key, "test");

```

Comment:
why not verifiedSet if you use later verifiedGet ? For first time run, it will most probably always pass as state is not there yet ?

Reply:
If you think a user should use only VerifiedGet and VerifiedSet then you are right. But that will have a big performance impact. The idea is to show a basic KV operation with or without authenticity check. What I don't understand is why this tiny detail is so important, because it is also found in the Java example. If this thing is so important, either let's fix in java or hide the java example altogether.

Reply:
The samples cover multiple functionalities; they are now described in the Readme.md